python/ray/data/tests/test_arrow_serialization.py

```diff
@@ -526,6 +541,22 @@ def fn(batch: list):
     assert res == [{"id": 1}], res
 
 
+def test_arrow_object_and_array_support(ray_start_regular_shared):
+    obj = ray.train.ScalingConfig(num_workers=4)
```

Comment:
I just needed an object that wasn't subject to the vagaries of the importing from the test module, so I chose this.